TFT/src/User/API/config.h

```diff
@@ -173,12 +174,8 @@ extern "C" {
 #define MIN_SIZE_LIMIT            -2000     // machine size less than this will not be parsed.
 #define NAME_MIN_LENGTH           3         // minimum name length
 #define GCODE_MIN_LENGTH          3         // gcode length less than this will not pe parsed.
-#if (IS_DELTA)
-  #define MIN_XY_POS_LIMIT        -2000     // Set a negative minimum position for Deltas
-#else
-  #define MIN_XY_POS_LIMIT        0         // position value less than this will not be parsed.
-#endif
-#define MIN_Z_POS_LIMIT           0
+#define MIN_XY_POS_LIMIT          -2000     // Set a negative minimum position for Delta printers
+#define MIN_Z_POS_LIMIT           -2000     // Set a negative minimum position for Delta printers
```

Comment:
Shouldn't we have 2 separate minimum z limits? This variable is used as a sanity check for the amount that the nozzle should be raised by when pausing, which should never be negative. It's also used with this stuff, which I'm not exactly sure how that works, but seems like it shouldn't ever be negative either.

https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/blob/bff5d814241e4f282bfb7fd14d7ba2a49687a348/TFT/src/User/API/config.c#L798-L808
&nbsp;
&nbsp;
Also, with `MIN_XY_POS_LIMIT`, for Cartesian printers this shouldn't be negative, so if the sanity check allows it to be negative for those printers, what's the point of the sanity check in the first place? Due to how different Deltas and Cartesian printers handle the X/Y values and where 0,0 is on both, I think it might be best to branch the sanity checks to have separate limits for these 2 different types of printers.

Reply:
we can do it easy without too much constraints (compile directives or new conf. param to distinguish cartesian or delta). It could be simply put a note in config.ini reporting that negative values can be used for delta as I did in the new param "x_raise_probing" I provided